### PR TITLE
giving kibana user privileges to create custom link index

### PR DESCRIPTION
**Pull request by cauemarcondes**
Addresses elastic/kibana#59305 by giving the kibana user reserved role privileges on `.apm-custom-link` to create APM custom link index in Kibana.

### CODE REVIEW

**Review by sorenlouv — APPROVED** [review]
lgtm (you'll need someone from security to sign-off as well)

**Review by legrego — COMMENTED** [review]
The privilege descriptor looks fine to me, but I'll defer to es-security for approval

**Review by tvernum — APPROVED** [review]

**Comment by cauemarcondes** [comment]
@tvernum could you tell me how to run the backport tool on ES, please?

**Pull request merged**
